### PR TITLE
[JA] Installation

### PR DESCRIPTION
**Pull request by unlimish**
Added translation Japanese

### CODE REVIEW

File: wiki/Installation/ja.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    -   インストール先を変更する場合、「Hi!」の下にある文字をクリックして、インストール先を変更します。
+    -   10秒後osu!はインストールされます。
+4.  インストール完了後、osu!はすぐに起動されます。osu!directは起動後にスターター譜面をダウンロードします。
+5.  osu!がサインインまたはアカウントを作成するように促してきます。
+    -   もし、アカウントを持っている場合は、サインインしてください。
+    -   もし、アカウントを作成したい場合は[アカウントを作成](/wiki/Registration)し、サインインしてください。
+
+## 譜面セットを追加
+
+*See also: [譜面](/wiki/Beatmaps) and [譜面セット](/wiki/Beatmapsets)*
```
Comment (cl8n) [inline wiki/Installation/ja.md]: any reason that this is not translated? (same w/ the "Note:" later on)

File: wiki/Installation/ja.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+譜面を追加するにはWebサイトから追加する方法と、osu!direct（[osu!supporter] (/wiki/osu!supporter)になる必要があります）からダウンロードする2つの方法があります。
```
Comment (cl8n) [inline wiki/Installation/ja.md]:
```suggestion
譜面を追加するにはosu!サイトから追加する方法と、osu!direct（[osu!supporter](/wiki/osu!supporter)になる必要があります）からダウンロードする2つの方法があります。
```

File: wiki/Installation/ja.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+2.  プレイしたい曲を探します。
+3.  あなたのコンピューターにダウンロードするために、ダウンロードアイコンをクリックします。
+4.  配置し、拡張子が.oszの譜面セットを開きます。
+    -   代わりに譜面セットをosu!がインストールされたディレクトリの「Songs」に配置しても譜面を追加できます。
+5.  osu!は自動的に譜面セットを処理して読み込みます。
+    -   なんらかの理由で読み込まれない場合は譜面の再読み込みをするために、[ソロモード](/wiki/Solo_mode)内で「F5」を押します。
+
+### osu!direct
+
+*Note: osu!サポータータグとosu!directを利用するためにインターネット接続が必要です。*
```
Comment (lolita-is-godddd) [inline wiki/Installation/ja.md]: I think it should be `osu!directを利用するためにosu!サポータータグとインターネット接続が必要です。` to make more clearly

Comment (unlimish) [reply wiki/Installation/ja.md]: LGTM It’s no problem my translation

Comment (cl8n) [reply wiki/Installation/ja.md]: you should apply this suggestion; currently it sounds as if you can *use* an osu!supporter tag, but the osu!supporter tag just lets you use osu!direct

Comment (unlimish) [reply wiki/Installation/ja.md]: I corrected the Note.
sorry, I didn't notice that before used grammar was strange...

**Comment by TPGPL** [comment]
could you translate "See also" and "Note" please?

File: wiki/Installation/ja.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+3.  スキンの製作者によってパッケージの形が違います。それに応じて適切な方法を選んでください。
+    -   スキンの拡張子が「.osk」の場合:
+        1.  osu!にスキンをドラッグ&ドロップする。
+        2.  osu!が自動的にスキンを読み込みそのスキンを適用します。
+    -   スキンが圧縮されている場合（「.7z」、「.rar」、「.zip」など):
+        1.  スキンを展開します。
+        2.  展開したスキンのフォルダをosu!のインストールされた場所にある「Skins」に移動します。
+        3.  osu!がすでに起動している場合は、新しいスキンを適用する前に再起動してください。
+        4.  osu!内の[オプション](/wiki/Options)を開き、「Skin」と検索します。
+        5.  「現在のスキン」までスクロールし、ダウンロードした新しいスキンを洗濯します。
```
Comment (kawaemon) [inline wiki/Installation/ja.md]: It should be "ダウンロードした新しいスキンを選択します". (I think that is conversion mistake.)
```suggestion
        5.  「現在のスキン」までスクロールし、ダウンロードした新しいスキンを選択します。
```

**Comment by TPGPL** [comment]
revert the `Refactored end of each sentence` commit asap

**Comment by unlimish** [comment]
> revert the `Refactored end of each sentence` commit asap

I did it
I'm sorry to make a conflict...X(

**Pull request merged**